### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by stramel** [angular-ui/AngularJS-Atom#21]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `angularjs` package
```
Package.getStylesheetsPath (/usr/local/share/atom/resources/app/src/package.js:448:9)
Package.getStylesheetPaths (/usr/local/share/atom/resources/app/src/package.js:459:32)
```

**Atom:** 0.189.0-edd46e4

**Comment by outsideris** [comment]
Thank you for reporting. I will check it.

**Comment by outsideris** [comment]
It's fixed in @716739a
